src/ui/components/AppWrapper/coreEventListeners.ts

```diff
@@ -42,11 +40,10 @@ const signifyOperationStateChangeHandler = async (
 ) => {
   switch (opType) {
   case OperationPendingRecordType.Witness:
-  case OperationPendingRecordType.Group:
     dispatch(updateIsPending({ id: oid, isPending: false }));
     dispatch(setToastMsg(ToastMsgType.IDENTIFIER_UPDATED));
     break;
-  case OperationPendingRecordType.Individual:
+  case OperationPendingRecordType.Group:
```

Comment:
We had
```
  case OperationPendingRecordType.Witness:
  case OperationPendingRecordType.Group:
    dispatch(updateIsPending({ id: oid, isPending: false }));
    dispatch(setToastMsg(ToastMsgType.IDENTIFIER_UPDATED));
    break;
```
which was effectively the same. Please revert the change here - the code is the same, so no need to duplicate.